docs/operators.md

```diff
@@ -104,9 +104,54 @@ LHS: left hand side, RHS: right hand side
 Logical operators apply to all types and yield a result of the same type as the
 operands. The right operand is evaluated conditionally.
 
-| Symbol | Operation                                           |
-|:------:|-----------------------------------------------------|
-|   &&   | Logical AND  `p && q  is  "if p then q else false"` |
-|   \|\| | Logical OR  `p \|\| q  is  "if p then true else q"` |
+| Symbol | Operation                                                                                                                                                                                                                     |
+|:------:|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|   &&   | Logical AND  `p && q  is  "if p then q else false"`                                                                                                                                                                           |
+|  \|\|  | Logical OR  `p \|\| q  is  "if p then true else q"`                                                                                                                                                                           |
+|   ??   | Logical NULLISH. The nullish coalescing (??) operator is a logical operator that returns its right-hand side operand when its left-hand side operand is null or undefined, and otherwise returns its left-hand side operand.  |
```

Comment:
Line says `... null or undefined` but we don't have null in uGO, so we need to remove `null` word here. And we need to say something like this after the explanation `As nullish term is widely used in programming, we preferred the same name although uGO doesn't have null values.`